cognite/client/utils/_time.py

```diff
@@ -60,6 +60,10 @@ def _import_zoneinfo_not_found_error() -> type[ZoneInfoNotFoundError]:
     return ZoneInfoNotFoundError
 
 
+def _datetime_is_tz_naive(dt: datetime) -> bool:
+    return dt.tzinfo is None or dt.tzinfo.utcoffset(dt) is None
```

Comment:
Feel free to remove the leading underscore, this lives in `_time.py` so everything is implied internal/private already (those public we explicitly expose)